apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedissonPluginConfig.java

```diff
@@ -40,6 +40,14 @@ public static class Redisson {
              * Set a negative number to save specified length of parameter string to the tag.
              */
             public static int REDIS_PARAMETER_MAX_LENGTH = 128;
+            /**
+             * If set to false, the PING command would be collected.
+             */
+            public static boolean PING_IGNORED = true;
```

Comment:
Why PING_IGNORED should be show by default? Is PING COMMAND having a special meaning?

Reply:
PING command will be ignored by default, PING in order to check if the channel is still connected(default 30 per second since 3.13.6, also can disabled).

Reply:
I think you should keep the configuration style consistent. Now, one configuration is with `show...`, the other is `ignored...`.  I think the style should use `show_xxx` and they should be aligned.